src/inline.d

```diff
@@ -1207,7 +1207,11 @@ public:
              * Note that inline scanning of s.exp.e1 and s.exp.arguments was already done.
              */
             if (s.exp && s.exp.op == TOKcall)
-                visitCallExp(cast(CallExp)s.exp, null, true);
+            {
+                CallExp ce = cast(CallExp)s.exp;
+                if (ce.e1.op == TOKvar)     // workaround for Bugzilla 15296
+                    visitCallExp(ce, null, true);
```

Comment:
OK, I added more detailed comment.

As I wrote, the essential problem is a `inlineScan` call in a few lines before. First [I tried to fix it](https://github.com/9rnsr/dmd/commits/fix15296b), but it was not possible quickly.